repo-validation/config/config_test.go

```
@@ -0,0 +1,390 @@
+	tempDir, err := os.MkdirTemp("", "config-test")
+	if err != nil {
+		t.Fatalf("Failed to create temp dir: %v", err)
+	}
+	defer os.RemoveAll(tempDir)
+
+	// Change to temp directory
+	oldWd, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Failed to get working directory: %v", err)
+	}
+	defer os.Chdir(oldWd)
```

golangci-lint (1.64.8)

Error return value of `os.Chdir` is not checked

(errcheck)